src/Dialogs/AliasDialog/AliasDialog.vala

```diff
@@ -92,27 +89,23 @@ public class Mail.AliasDialog : Hdy.ApplicationWindow {
         var frame = new Gtk.Frame (null) {
             margin_start = 12,
             margin_end = 12,
-            margin_bottom = 12,
             child = content_box
         };
 
-        var box = new Gtk.Box (VERTICAL, 0);
-        box.add (header);
-        box.add (frame);
-
         toast = new Granite.Widgets.Toast ("");
         toast.set_default_action (_("Undo"));
 
         var overlay = new Gtk.Overlay () {
-            child = box
+            child = frame
         };
         overlay.add_overlay (toast);
+        overlay.show_all ();
 
         title = _("Aliases");
         default_height = 300;
         default_width = 500;
-        add (overlay);
-        show_all ();
+        get_content_area ().add (overlay);
+        this.add_button (_("Close"), Gtk.ResponseType.CLOSE);
         present ();
```

Comment:
We probably should call `present ()` after creating the dialog and not in `construct` otherwise I get this warning:
```
(io.elementary.mail:64822): Gtk-WARNING **: 17:56:20.812: gtk_window_set_titlebar() called on a realized window
```

Reply:
Nice catch